src/main/java/core/basesyntax/operation/Operation.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.operation;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
The `code` variable can be made final since it is not expected to change after the enum constant is created.